README.md

```diff
@@ -143,3 +144,25 @@ It's also possible to run the script without using docker. To this end, issue th
     export ROS_WS=/tmp/ros_ws        # define a new ROS workspace location
     mkdir $ROS_WS                    # and create it
     .moveit_ci/travis.sh
+
+## Enabling codecov.io reporting
+
+For codecov to work you need to build and link your C++ code with specific parameters.  To enable this we use the ROS package [code_coverage](https://github.com/mikeferguson/code_coverage).  To to use the `code-coverage` test in your repo make these two changes:
+
+1. Add `<test_depend>code_coverage</test_depend>` to your package.xml
+1. Add this to your `CMakeLists.txt`:
+
+```cmake
+# to run: catkin_make -DENABLE_COVERAGE_TESTING=ON package_name_coverage
+if(CATKIN_ENABLE_TESTING AND ENABLE_COVERAGE_TESTING)
```

Comment:
Maybe split up this if condition into two separate ones to match your code snippet below?

Reply:
This is the exact code that is in the moveit codebase already.  I just copied it from there.  What I did in the unit test is the minimal amount needed.  What is kind of nice about this if statement (like what is in the moveit codebase) is it adds the codecoverage to one section with out being two blocks of code like what's in the example from code_coverage ros package's README.md.

Reply:
This is coppied code from the cmake files in moveit.  I like how it keeps the code coverage section separate from the testing if section.  I can change it if you'd perfer.  I made the unit test match this.